db/migrations/0003_alter_movie_actors_alter_movie_genres_and_more.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 5.1.4 on 2024-12-30 15:37
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_cinemahall_movie_moviesession'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='actors',
+            field=models.ManyToManyField(related_name='actors', to='db.actor'),
```

Comment:
The `related_name` for the `actors` field is set to 'actors', which might cause confusion as it is the same as the field name. Consider using a more descriptive related name to avoid potential conflicts.